frontend/src/UploadCsv.js

```diff
@@ -37,7 +37,15 @@ function UploadCsvPage(props) {
           console.log(res.data.errors)
           console.log('full response: ')
           console.log(res)
+        } else {
+          console.log('Unexpected *response* from server: ')
+          console.log(res)
         }
+      }).catch((err) => {
```

Comment:
FYI: I updated Axios to not just use `.then()` to also not just respond to `200`'s and print their error messages (which is how I was returning them before because I didn't know how to show otherwise). I also added `.catch()` to properly handle errors as well.